timed/employment/admin.py

```diff
@@ -1,10 +1,66 @@
 """Views for the admin interface."""
 
+import datetime
+
+from django import forms
 from django.contrib import admin
+from django.core.exceptions import ValidationError
 
 from timed.employment import models
 
 
+class EmploymentForm(forms.ModelForm):
+    """Custom form for the employment admin."""
+
+    def clean(self):
+        """Validate the employment as a whole.
+
+        Ensure the end date is after the start date and there is only one
+        active employment per user and there are no overlapping employments.
+
+        :throws: django.core.exceptions.ValidationError
+        :return: The cleaned data
+        :rtype:  dict
+        """
+        data = super().clean()
+
+        employments = models.Employment.objects.filter(user=data.get('user'))
+
+        if (
+            data.get('end_date') and
+            data.get('start_date') >= data.get('end_date')
+        ):
+            raise ValidationError('The end date must be after the start date')
```

Comment:
This is the first string introduced in the timed backend... even if timed is not internationalized at the time wouldn't it make sense to use the translate module here?